data/hai/unfettered side missions.txt

```diff
@@ -0,0 +1,533 @@
+			scene "scene/cave scene"
+			`	The second one now takes a step toward you to speak, almost interrupting the first. "We have a proposition for you: should you agree to bring much needed supplies to our brethren, the Unfettered Hai, we will sell you this ship for the modest price of 10 million credits, including 2 very powerful Korath Detainers!" The first one seems surprised by this and tries to intervene, but the second resumes before they can speak up. "These legendary weapons can interfere with nearly any enemy engine system to give you a significant maneuverability advantage. Worry not, you'll be able to recuperate the cost of this ship with ease by bringing medical supplies to the Unfettered. This cargo is needed on short notice, on <destination>, and we lack the time and personnel to bring them.`
+			`	"Of course, the ship is aged, and you should not expect the best price in a shipyard. Nevertheless, it is a fine vessel, capable of fighting, freighting, and mining. So, what do you say, <last>? This is the opportunity of a lifetime; they simply don't make ships like this anymore."`
+			label choice
+			choice
+				`	"I'll take the... Anomalocaris, was it?"`
+					goto take
+				`	"If you lack crew, who was piloting the ship from the ocean?"`
+					to display
+						not "unfettered: pilot dialog"
+					goto pilot
+				`	"Why was the security ship submerged?"`
+					to display
+						not "unfettered: ocean dialog"
+					goto ocean
+				`	"No thanks. I'm not looking to add another ship to my fleet."`
+			`	"A shame. We will probably not meet again. Farewell," one says. They signal the ship to submerge into the ocean again and walk off along the beach.`
+				goto trainback
+			label pilot
+			action
+				set "unfettered: pilot dialog"
+			`	"Actually, it's a 'what,' not a-" starts the first one, but they are again interrupted by the second.`
+			`	"The less you know, the better for all of us, Captain."`
+				goto choice
+			label ocean
+			action
+				set "unfettered: ocean dialog"
+			`	"We found it there," the first one answers.`
+			`	"It is hard to find a drop of rain in a storm, but the drop of rain may see all, and we were that drop. Worry not, we have restored its hull," the other one adds.`
+				goto choice
+			label take
+			action
+				give ship "Anomalocaris (Venerable)" "Good Omen"
+					id "hai legacy"
+				fine 10000000
+				log "Accepted a somewhat shady deal for an old Hai ship, an Anomalocaris, that seems to still be in good functioning state."
+			`	They give you access to the ship and proceed to explain the details. "It is of no real significance, but surely you understand we cannot go through standard means for this sale. We will instead present it as a fine. You'll be charged for carrying Korath tech you had no idea was forbidden, and it will not go on your permanent record. These Detainers are suspicious looking indeed."`
```

Comment:
```suggestion
			`	They give you access to the ship and proceed to explain the details. "It is of no real significance, but surely you understand we cannot go through standard means for this sale. We will instead present it as a fine. You'll be charged for carrying Korath tech you had no idea was forbidden, and it will not go on your permanent record. These Detainers are suspicious-looking indeed."`
```